uber/site_sections/registration.py

```diff
@@ -160,9 +160,7 @@ def delete(self, session, id, return_to='index?'):
                 session.add(Attendee(**{attr: getattr(attendee, attr) for attr in [
                     'group', 'registered', 'badge_type', 'badge_num', 'paid', 'amount_paid', 'amount_extra'
                 ]}))
-                session.assign_badges(attendee.group, attendee.group.badges + 1, attendee.badge_type)
                 session.delete_from_group(attendee, attendee.group)
-                attendee.group.attendees.remove(attendee)
```

Comment:
This line already shows up in the ``session.delete_from_group()`` function which is called on the previous line. Because it's already been called, this attendee no longer has a group set, which is what caused us to raise an ``AttributeError``.